spec/duration.html

```diff
@@ -448,7 +448,7 @@ <h1>Temporal.Duration.prototype.round ( _options_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-temporal.duration.prototype.total">
-      <h1>Temporal.Duration.prototype.total ( _options_ )</h1>
+      <h1>Temporal.Duration.prototype.total ( [ _options_ ] )</h1>
```

Comment:
This one is not optional because the `unit` is required. (Consider adding an `undefined` -> TypeError check in #1646?)